pkg/types/installer.go

```diff
@@ -13,3 +15,41 @@ type InstallerSpec struct {
 	CreatedAt       util.Time `json:"created"`
 	Immutable       bool      `json:"isInstallerNotEditable"`
 }
+
+type InstallerSpecResponse struct {
+	Body InstallerSpec `json:"installer"`
+}
+
+type CreateInstallerRequest struct {
+	Yaml string `json:"yaml"`
+}
+
+type PromoteInstallerRequest struct {
+	Sequence     int64  `json:"sequence"`
+	VersionLabel string `json:"versionLabel"`
+	ChannelID    string `json:"channelId"`
+}
+
+type EnterpriseChannel struct {
```

Comment:
why are these here? graphql requests don't interact with the enterprise api

Reply:
they were provisional during dev, forgot to remove them after I found they weren't needed.  Removed now.